src/app/modules/main/chat_section/chat_content/module.nim

```diff
@@ -185,7 +185,8 @@ proc buildPinnedMessageItem(self: Module, messageId: string, actionInitiatedBy:
       transactionValue,
       m.transactionParameters.transactionHash,
       m.transactionParameters.commandState,
-      m.transactionParameters.signature)
+      m.transactionParameters.signature),
+      m.mentionedUsersPks
```

Comment:
also this is outside the bracket? somethings off I think.

Reply:
This bracket belongs to the end of newTransactionParametersItem function.
The end of initItem bracket is below.